src/views/Personalisation.vue

```diff
@@ -0,0 +1,316 @@
+<template>
+    <div class="bg-white rounded p-6">
+        <div class="flex items-center mb-8">
+            <h1 class="text-4xl">
+                Behavioral tracking
+            </h1>
+        </div>
+
+        <hr class="mb-6">
```

Comment:
Kig på hvordan det er gjort på settings og align brugen af hr's.

Reply:
Er du sikker? Jeg synes at my-10 er lidt for meget.
![image](https://github.com/Relewise/relewise-demos-shop-vue/assets/141124692/86c79e82-919d-4f54-ab74-5001669c12c4)

Reply:
Align the UI with Settings Page